indigo compile failure

I'll handle this

```
[ 74%] [ 75%] Building CXX object CMakeFiles/jsk_pcl_ros.dir/src/pointcloud_moveit_filter.cpp.o
Building CXX object CMakeFiles/jsk_pcl_ros.dir/src/intermittent_image_annotator_nodelet.cpp.o
In file included from /home/k-okada/catkin_ws/ws_jsk_recognition/src/jsk_recognition/jsk_pcl_ros/src/pointcloud_moveit_filter.cpp:36:0:
/home/k-okada/catkin_ws/ws_jsk_recognition/src/jsk_recognition/jsk_pcl_ros/include/jsk_pcl_ros/pointcloud_moveit_filter.h: In member function void jsk_pcl_ros::PointCloudMoveitFilter::cloudMsgCallback(const ConstPtr&):
/home/k-okada/catkin_ws/ws_jsk_recognition/src/jsk_recognition/jsk_pcl_ros/include/jsk_pcl_ros/pointcloud_moveit_filter.h:131:55: error: no matching function for call to point_containment_filter::ShapeMask::maskContainment(pcl::PointCloud<pcl::PointXYZ>&, Eigen::Vector3d&, double, double&, std::vector<int>&)
                                      max_range_, mask_);
                                                       ^
/home/k-okada/catkin_ws/ws_jsk_recognition/src/jsk_recognition/jsk_pcl_ros/include/jsk_pcl_ros/pointcloud_moveit_filter.h:131:55: note: candidate is:
In file included from /home/k-okada/catkin_ws/ws_jsk_recognition/src/jsk_recognition/jsk_pcl_ros/include/jsk_pcl_ros/pointcloud_moveit_filter.h:46:0,
                 from /home/k-okada/catkin_ws/ws_jsk_recognition/src/jsk_recognition/jsk_pcl_ros/src/pointcloud_moveit_filter.cpp:36:
/opt/ros/indigo/include/moveit/point_containment_filter/shape_mask.h:85:8: note: void point_containment_filter::ShapeMask::maskContainment(const PointCloud2&, const Vector3d&, double, double, std::vector<int>&)
   void maskContainment(const sensor_msgs::PointCloud2& data_in,  const Eigen::Vector3d &sensor_pos,
        ^
/opt/ros/indigo/include/moveit/point_containment_filter/shape_mask.h:85:8: note:   no known conversion for argument 1 from pcl::PointCloud<pcl::PointXYZ> to const PointCloud2& {aka const sensor_msgs::PointCloud2_<std::allocator<void> >&}
make[2]: *** [CMakeFiles/jsk_pcl_ros.dir/src/pointcloud_moveit_filter.cpp.o] Error 1
```

--- a/jsk_pcl_ros/include/jsk_pcl_ros/pointcloud_moveit_filter.h
+++ b/jsk_pcl_ros/include/jsk_pcl_ros/pointcloud_moveit_filter.h
@@ -42,6 +42,7 @@
 #include <tf/message_filter.h>
 #include <message_filters/subscriber.h>
 #include <sensor_msgs/PointCloud2.h>
+#include <moveit/version.h>
 #include <moveit/occupancy_map_monitor/occupancy_map_updater.h>
 #include <moveit/point_containment_filter/shape_mask.h>
 #include <moveit/occupancy_map_monitor/occupancy_map_monitor.h>
@@ -127,8 +128,13 @@ namespace jsk_pcl_ros
           return;
         }
         /* mask out points on the robot */
+#if (MOVEIT_VERSION_MINOR < 6)
         shape_mask_->maskContainment(xyz_cloud, sensor_origin_eigen, 0.0,
                                      max_range_, mask_);
+#else   // from moveit 0.6 (indigo), PCL dependency is removed
+        shape_mask_->maskContainment(*cloud_msg, sensor_origin_eigen, 0.0,
+                                     max_range_, mask_);
+#endif
         typename pcl::PointCloud<PointT>::Ptr
           filtered_cloud (new pcl::PointCloud<PointT>);
         pcl::PointIndices::Ptr indices (new pcl::PointIndices);

